packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -290,7 +284,7 @@ export default class GoogleCalendarService implements Calendar {
           eventId: event.id || "",
           requestBody: {
             description: getRichDescription({
-              ...formattedCalEvent,
+              ...calEvent,
               additionalInformation: { hangoutLink: event.hangoutLink },
```

Comment:
The only time we need to call `getRichDescription` is when we're adding a Meet link